semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -184,7 +184,8 @@ public static boolean isAbsoluteUrl(String urlStr) {
    */
   public static Optional<String> getHttpUrlForOTelFormat(
       Map<String, AttributeValue> attributeValueMap) {
-    if (attributeValueMap.containsKey(HTTP_URL.getValue())) {
+    if (attributeValueMap.containsKey(HTTP_URL.getValue())
+        && isAbsoluteUrl(attributeValueMap.get(HTTP_URL.getValue()).getValue())) {
```

Comment:
What if this is not an absolute url but we don't have enough attributes to build the url as well?

Reply:
Fallback to present case only?